apps/ex_nvr_web/lib/ex_nvr_web/live/recordings_list_live.ex

```diff
@@ -141,15 +141,29 @@ defmodule ExNVRWeb.RecordingListLive do
        devices: Devices.list(),
        popup_open: false,
        filter_params: params,
-       pagination_params: %{}
+       pagination_params: %{},
+       sort_params: %{}
      )}
   end
 
   @impl true
   def handle_params(params, _uri, socket) do
+    sort_params = Map.take(params, ["order_by", "order_directions"])
+
     case Recordings.list(params) do
-      {:ok, {recordings, meta}} ->
-        {:noreply, assign(socket, meta: meta, recordings: recordings)}
+      {:ok, {recordings, %{total_pages: total_pages} = meta}} ->
+        current_page = Map.get(params, "page", "0") |> String.to_integer()
+
+        if current_page > total_pages do
```

Comment:
Is there a reason to add this validation ? it's not a problem if the user enter a non valid page number.